index.html

```diff
@@ -1,103 +1,137 @@
+           <li><a href="#about">About</a></li>
+           <li><a href="#experience">Experience</a></li>
+           <li><a href="#skills">Skills</a></li>
+        </ul>
+      </nav>
+    </header>
+  </div>
+  
+  <section id="about">
+     <h1>ABOUT</h1>
+     <img id="img-riri" src="images/riri.png" alt="riri">
+     <p id="about-me">Hi, My name is  Raíssa, I was born and raised in São Paulo Brazil. I moved to the USA in 2013, I now live in Chicago. I'm currently a student at Code the Dream, I am learning front-end development skills, and I hope to one day become a Software Developer.</p>  
+  </section>
+
+
+  <section id="experience">
+     <h1>EXPERIENCE</h1>
+     <table>
+      <tr>
+        <th>Role</th>
+        <th>Company</th>
+        <th>Date</th>
+      </tr>
+      
+      <tr>
+        <td>Data Resercher</td>
+        <td>Human Predictions</td>
+        <td>12/2020 - 09/2022</td>
+      </tr>
+      
+      <tr>
+        <td>Ecommerce Stylist</td>
+        <td>Notre Shop</td>
+        <td> 03/2018 - 03/2020</td>
+      </tr>
+    
+     <tr>
+        <td>Sales Associate</td>
+        <td>Fishmen Fabrics</td>
+        <td> 07/2017 - 03/2018</td>
+      </tr>
+    </table>
 
-</section>
-
-
-<section id="connect">
-    <h2>Connect</h2>
-
-      <ul>
-         <li>raissanfg@gmail.com</li>
-         <li>https://twitter.com/raissanfg</li>
-         <li>https://www.instagram.com/raissanfg</li>
-      </ul>
-
-</section>
-
-<section>
-   <h2>Leave a Message</h2>
-     <form name="leave_message"> 
-
-      <label class="field_label" for="name">Name</label><br>
-      <input type="text" name="name" required="true"><br>
-
-      <label class="field_label" for="email">Email</label><br>
-      <input type="email" name="email" required="true"><br>
-
-      <label class="field_label" for="message">Message</label><br>
-      <textarea name="message" required="true"></textarea> <br>
+  </section>
 
-     <button type="submit">Submit</button><br>
+  <section id="skills">
+    <h1>SKILLS</h1>
+    <ul>
+      <li class="skl">HTML</li>
+      <li class="skl">CSS</li>
+      <li class="skl">JAVASCRIPT</li>
+      <li class="skl">GIT</li> 
+    </ul>
+  </section>
+
+  <section id="leave_message">
+   <h1>LEAVE A MESSAGE!</h1>
+     <form name="leave-a-message"> 
+      
+        <label for="name">Name</label><br>
+        <input type="text" id="name" name="name" required="true"> 
+      
+        <label for="email">Email Address</label><br>
+        <input type="email" id="email" name="email" required="true"> 
+      
+        <label for="message">Message</label><br>
+        <textarea type="message" id="message" name="message" required="true"></textarea><br>
+      
+         <button id="button" type="submit">Submit</button><br> 
 
      </form>
-</section>
-
-<section id="messages">
-  <h2>Messages</h2>
-   <ul>
+  </section>
 
-   </ul>
-
-
-</section>
+  <section id="messages">
+     <h1 id="message-box">MESSAGES</h1>
+       <ul>
```

Comment:
It would be good to add some styling to this section. Currently it's a bit difficult to read with the user's name being in Blue and the message being in Black text on a black background:

![](https://user-images.githubusercontent.com/9094098/210592301-c0a58ef5-e5d1-41b9-bd1f-68a65f22fa77.png)